src/instructlab/config/init.py

```diff
@@ -111,20 +111,21 @@ def init(
     elif interactive:
         entries = listdir(DEFAULTS.TRAIN_PROFILE_DIR)
         click.echo("Please choose a train profile to use:")
+        click.echo("[0] No profile (CPU-only)")
         for i, value in enumerate(entries):
-            click.echo(f"{i}. {value}")
+            click.echo(f"[{i+1}] {value}")
         train_profile_selection = click.prompt(
-            "Enter the number of your choice [hit enter for the default]",
+            "Enter the number of your choice [hit enter for the default CPU-only profile]",
             type=int,
-            default=-1,
+            default=0,
         )
-        if 0 <= train_profile_selection < len(entries):
-            click.echo(f"You selected: {entries[train_profile_selection]}")
+        if 1 <= train_profile_selection < len(entries):
+            click.echo(f"You selected: {entries[train_profile_selection+1]}")
             cfg.train = read_train_profile(
-                join(DEFAULTS.TRAIN_PROFILE_DIR, entries[train_profile_selection])
+                join(DEFAULTS.TRAIN_PROFILE_DIR, entries[train_profile_selection + 1])
```

Comment:
I think this should be -1?

Reply:
Hmmm, the example output would suggest otherwise but I wouldn't mind a sanity check, I don't want to cause a regression

Reply:
Oh I think you're right! Good catch!

Reply:
Changed both to `- 1` and looks like that fixed it
```bash
Please choose a train profile to use:
[0] No profile (CPU-only)
[1] A100_H100_x4.yaml
[2] A100_H100_x8.yaml
[3] A100_H100_x2.yaml
[4] L40_x8.yaml
[5] L40_x4.yaml
[6] L4_x8.yaml
Enter the number of your choice [hit enter for the default CPU-only profile] [0]: 3
You selected: A100_H100_x2.yaml
Initialization completed successfully, you're ready to start using `ilab`. Enjoy!
```